docs/basic-features/image-optimization.md

```diff
@@ -121,6 +121,8 @@ The following Image Optimization cloud providers are included:
 - [Imgix](https://www.imgix.com): `loader: 'imgix'`
 - [Cloudinary](https://cloudinary.com): `loader: 'cloudinary'`
 - [Akamai](https://www.akamai.com): `loader: 'akamai'`
+- Static: uses image from url mentioned in `src` parameter without optimization
+  `loader: 'static'`
```

Comment:
```suggestion
- dangerously-unoptimized: uses image from url mentioned in `src` parameter without optimization (only needed with `next export` or when the built-in `_next/image` optimizer can not be leveraged)
  `loader: 'dangerously-unoptimized'`
```

Let's rename this to `dangerously-unoptimized` to stress the image will no longer be automatically optimized when this loader is leveraged.